lab-yunjoo/test/note-route-test.js

```diff
@@ -0,0 +1,156 @@
+'use strict';
+
+const request = require('superagent');
+const expect = require('chai').expect;
+const server = require('./../server');
+const port = process.env.PORT || 3000;
+const serverUrl = `http://localhost:${port}`;
+
+describe('testing note-route module', function(){
```

Comment:
This describe block doesn't do anything as written. If you had more than one test file that interacted with your server, all but the first one would break. 

This describe block, which starts your server before and closes it after, doesn't have any 'it' blocks inside of it, so it will never run the before block or the after block. 

Basically, what I'm saying is that all of the other code in this file should have been nested inside of this describe block.